request-manager/graph/resolver.go

```diff
@@ -608,23 +585,28 @@ func (o *resolver) newNoopNode(name string, jobArgs map[string]interface{}) (*No
 				id: jid,
 			}
 		default:
-			return nil, fmt.Errorf("Error making no-op node %s: %s", name, err)
+			return nil, fmt.Errorf("Error making no-op job %s: %s", name, err)
 		}
 	}
 	err = rj.Create(jobArgs)
 	if err != nil {
-		return nil, fmt.Errorf("Error creating no-op node %s: %s", name, err)
+		return nil, fmt.Errorf("Error creating no-op job %s: %s", name, err)
+	}
+	bytes, err := rj.Serialize()
+	if err != nil {
```

Comment:
Here's a great example of why I suggest `if err := func(); err != nil {` (i.e. block-scoping `err`): is `err` on this line from `bytes, err := rj.Serialize()` or `err = rj.Create(jobArgs)` just above? 🤔